lib/premailer/rails/css_loaders/network_loader.rb

```diff
@@ -14,7 +14,7 @@ def uri_for_url(url)
 
           if uri.host.present?
             return uri if uri.scheme.present?
-            URI("http://#{uri.to_s}")
+            URI::HTTP.build(port: uri.port, host: uri.host, path: uri.path)
```

Comment:
Wouldn't it be enough to simply drop the `//` from the string here, i.e. `URI("http:#{uri}")`?